src/components/RefinementList/RefinementList.js

```diff
@@ -127,6 +128,8 @@ class RefinementList extends React.Component {
     }
 
     const limit = this.state.isShowMoreOpen ? this.props.limitMax : this.props.limitMin;
+    this.props.facetValues = this.props.facetValues.slice(0, limit);
```

Comment:
Ok my bad, just saw the Travis tests. I'll fix it.

Can't we have an eslint rule to detect that somehow?